app/src/test/java/edu/uc/hickmadc/customnotifications/ExampleUnitTest.kt

```diff
@@ -1,8 +1,12 @@
 package edu.uc.hickmadc.customnotifications
```

Comment:
Rename the file, as it is actually performing tests now.  "Example" is no longer appropriate.